lib/hawkular/operations/operations_api.rb

```diff
@@ -105,7 +127,8 @@ def connect
         client.on(:message, once: true) do |msg|
           parsed_message = msg.data.to_msg_hash
 
-          logger.log("Sent WebSocket message: #{parsed_message}")
+          # Not working with websocket-vcr
```

Comment:
BTW, can we do the new instance of logger? Logging that message could be useful...